### PR TITLE
[IMP] CRM: rewrite gamification

### PR DESCRIPTION
**Pull request by jero-odoo**
Docs task: https://www.odoo.com/mail/view?model=project.task&res_id=3878226&access_token=9a8cabb2-d227-44ac-9fe0-8746d9193e87
Complete rewrite: https://www.odoo.com/documentation/17.0/applications/sales/crm/optimize/gamification.html

Updating images and content for V17
Update to content standards
Adding content for creating challenges and rewards

### CODE REVIEW

**Comment by jero-odoo** [comment]
Hi @hojo-odoo this is ready for review. Thanks!

File: content/applications/sales/crm/optimize/gamification.rst [hunk 1]
```diff
@@ -1,86 +1,159 @@
...
+CRM Gamification
+================
 
-Challenging your employees to reach specific targets with goals and rewards is an excellent way to
-reinforce good habits and improve your salespeople productivity. The Gamification module gives you
-simple and creative ways to motivate and evaluate your employees with real-time recognition and
-badges inspired by game mechanics.
+The *Gamification tools* provide the opportunity to evaluate and motivate users through customizable
+challenges, goals, and rewards. Goals are created to target actions within the *CRM* app, can be
+tracked and rewarded automatically.
```
Comment (hojo-odoo) [inline content/applications/sales/crm/optimize/gamification.rst:7]:
```suggestion
In Odoo's *CRM* app, *Gamification tools* provide the opportunity to evaluate and motivate users through customizable
challenges, goals, and rewards. Goals are created to target actions within the *CRM* app, and can be
tracked and rewarded automatically.
```

File: content/applications/sales/crm/optimize/gamification.rst [hunk 2]
```diff
@@ -1,86 +1,159 @@
...
+Create badges
+=============
 
-.. note::
-   Badges are granted when a challenge is finished. This is either at the end of a running period
-   (eg: end of the month for a monthly challenge), at the end date of a challenge (if no periodicity
-   is set), or when the challenge is manually closed.
+*Badges* are awarded to users when they have completed a challenge. Different badges can be awarded
+based on the type of task completed, and can be issued to more than one use depending on the time
+they accomplish the goal.
```
Comment (hojo-odoo) [inline content/applications/sales/crm/optimize/gamification.rst:46]:
```suggestion
based on the type of task completed, and can be issued to more than one user, depending on the time
they accomplish the goal.
```

File: content/applications/sales/crm/optimize/gamification.rst [hunk 3]
```diff
@@ -1,86 +1,159 @@
...
-------------
+.. note::
+   Some badges can be awarded outside of challenges, as well. Select the Kanban card for the desired
+   badge, then click :guilabel:`Grant`. This opens a :guilabel:`Grant Badge` pop-up window. Select
+   a user from the :guilabel:`Who would you like to reward?` field.
+
+   Add any additional information regarding why the user is receiving the reward in the field below,
+   then click :guilabel:`Grant Badge`.
+
+To create a new badge, click :guilabel:`New`, at the top-left of the page, to open a blank form.
```
Comment (hojo-odoo) [inline content/applications/sales/crm/optimize/gamification.rst:63]:
```suggestion
To create a new badge, click :guilabel:`New` at the top-left of the page to open a blank form.
```

File: content/applications/sales/crm/optimize/gamification.rst [hunk 4]
```diff
@@ -1,86 +1,159 @@
...
+Enter a name for the :guilabel:`Badge`, followed by a description.
+
+The :guilabel:`Allowance to Grant` field determines when a badge can be granted, and by whom.
```
Comment (hojo-odoo) [inline content/applications/sales/crm/optimize/gamification.rst:66]:
```suggestion
The :guilabel:`Allowance to Grant` field determines when a badge can be granted, and by whom:
```

File: content/applications/sales/crm/optimize/gamification.rst [hunk 5]
```diff
@@ -1,86 +1,159 @@
...
 
-For non-numerical achievements, **Badges** can be granted to users. From a simple thank you to an
-exceptional achievement, a badge is an easy way to express gratitude to a user for their good work.
-To create and grant badges to your employees based on their performance, go to
-:menuselection:`Settings --> Gamification Tools --> Badges`.
+Create assignement rules
+------------------------
 
-.. image:: gamification/badges.png
+To assign the challenge to specific users, one or more assignment rule must be utilized.
```
Comment (hojo-odoo) [inline content/applications/sales/crm/optimize/gamification.rst:101]:
```suggestion
To assign the challenge to specific users, one or more assignment rules must be utilized.
```

**Review by hojo-odoo — APPROVED** [review]
Hey @jero-odoo,

Great job on this doc. I'm approving with a few small suggestions.

Thanks,
hojo

**Comment by jero-odoo** [comment]
@StraubCreative this is ready for final. Thanks!

File: content/applications/sales/crm/optimize/gamification.rst [hunk 6]
```diff
@@ -1,86 +1,159 @@
...
+In Odoo's *CRM* app, *Gamification tools* provide the opportunity to evaluate and motivate users
+through customizable challenges, goals, and rewards. Goals are created to target actions within the
+*CRM* app, and can be tracked and rewarded automatically.
 
 Configuration
 =============
 
-Install the *Gamification* module, or the *CRM gamification* one, which adds some useful data
-(goals and challenges) that can be used on *CRM/Sale*.
+To install the *Gamification* module, navigate to the :menuselection:`Apps` application. Click into
```
Comment (StraubCreative) [inline content/applications/sales/crm/optimize/gamification.rst:12]: On a follow-up PR let's re-examine this section. It's not exactly clear how or when these apps are installed automatically so let's try to cover that.

File: content/applications/sales/crm/optimize/gamification.rst [hunk 7]
```diff
@@ -1,86 +1,159 @@
...
+
+The :guilabel:`Allowance to Grant` field determines when a badge can be granted, and by whom:
+
+- :guilabel:`Everyone`: this badge can be manually granted by any user.
+- :guilabel:`A selected list of users`: this badge can only be granted by a select group of users.
+  If this option is selected, it generates a new field, :guilabel:`Authorized Users`. Choose the
+  appropriate users from this drop-down list.
+- :guilabel:`People having some badges`: this badge can only be granted by users who have already
+  been awarded a specific badge. If this option is selected it generates a new field,
+  :guilabel:`Required Badges`. Use this drop-down to select the badges a user must have before they
```
Comment (StraubCreative) [inline content/applications/sales/crm/optimize/gamification.rst:74]: suggestion
```suggestion
  :guilabel:`Required Badges`. Use this drop-down list to select the badge(s) a user must have before they
```

File: content/applications/sales/crm/optimize/gamification.rst [hunk 8]
```diff
@@ -1,86 +1,159 @@
...
 
-Set up rewards
---------------
+At the top of the form, enter a :guilabel:`Challenge Name`.
 
-For non-numerical achievements, **Badges** can be granted to users. From a simple thank you to an
-exceptional achievement, a badge is an easy way to express gratitude to a user for their good work.
-To create and grant badges to your employees based on their performance, go to
-:menuselection:`Settings --> Gamification Tools --> Badges`.
+Create assignement rules
```
Comment (StraubCreative) [inline content/applications/sales/crm/optimize/gamification.rst:98]:
```suggestion
Create assignment rules
```

File: content/applications/sales/crm/optimize/gamification.rst [hunk 9]
```diff
@@ -1,86 +1,159 @@
...
+necessary, click into the third field to further define the parameter.
+
+.. tip::
+   To include all users with permissions in the *Sales* app, create a rule with the following
+   parameters:
+   - :guilabel:`Groups`
+   - :guilabel:`is in`
+   - `Sales/User: Own Documents Only`
+
+   .. image:: gamification/assignation-rule.png
```
Comment (StraubCreative) [inline content/applications/sales/crm/optimize/gamification.rst:114]: The way the condition is showing in the field is a display bug (it should be flush with the row), so let's retake and zoom out slightly or increase browser window size.

**Review by StraubCreative — APPROVED** [review]
Hi @jero-odoo 
Thanks for the rewrite! 
Approving with comments, which I'll push up most of the revisions for in another commit. For the remaining comments (config + screenshot retake), let's address on a new PR.

**Comment by StraubCreative** [comment]
@robodoo r+

**Pull request closed without merging**